agent-bridge/src/main/java/com/newrelic/agent/bridge/PublicApi.java

```diff
@@ -141,6 +141,14 @@
      */
     void addCustomParameter(String key, String value);
 
+    /**
+     * Add a key/value pair to the current transaction. These are reported in errors and transaction traces.
+     *
+     * @param key Custom parameter key.
+     * @param value Custom parameter value.
+     */
+    void addCustomParameter(String key, Boolean value);
```

Comment:
Can we simplify this to `addCustomParameter(String key, boolean value)` to avoid the case of somebody accidentally passing null?  This makes for a stronger API in my opinion.